python/plugins/db_manager/db_plugins/gpkg/connector.py

```diff
@@ -620,6 +620,20 @@ def runVacuum(self):
         """ run vacuum on the db """
         self._execute_and_commit("VACUUM")
 
+    def commentTable(self, schema, tablename, comment):
+        """Comment the table"""
+        return ''
+
+    def commentTable(self, schema, tablename, comment):
+        return ''
+
+    def unCommentTable(self, schema, tablename):
```

Comment:
I'd say a single `commentTable(self, schema, tablename, comment=None)` should be sufficient